app/code/community/Reverb/ReverbSync/Model/Mapper/Product.php

```diff
@@ -83,6 +84,7 @@ public function getCreateListingWrapper(Mage_Catalog_Model_Product $product)
         $name = $product->getName();
         $description = $this->getProductDescription($product);
         $sku = $product->getSku();
+        $sku = urlencode($sku);
```

Comment:
this feels a little strange to override this variable. isn't the only place you want to urlencode it in the GET lookup to reverb?